sequence_transform.c

```diff
@@ -297,7 +297,7 @@ void st_find_missed_rule(void)
     st_trie_rule_t result;
     result.sequence = sequence_str;
     result.transform = transform_str;
-    const int new_len = st_trie_get_rule(&trie, &key_buffer, search_len_start, &result);    
+    const int new_len = st_trie_get_rule(&trie, &key_buffer, search_len_start, &result);
```

Comment:
Removing this white space was not an intentional change. I wonder what is causing my editor to do this light weight linting.